app/services/spree/netsuite_order_service.rb

```diff
@@ -24,7 +24,7 @@ def create_order(order, items)
         },
         location: { id: @location_id },
         orderstatus: payment[:status],
-        shipmethod: { id: 66857 },
```

Comment:
need to dynamic this value expose new field on Netsuite settings on admin pannel. "shipping_method_id"